types/wordpress__e2e-test-utils-playwright/index.d.ts

```diff
@@ -0,0 +1,52 @@
+declare module '@wordpress/e2e-test-utils-playwright' {
+	import type { Browser, Page, BrowserContext } from '@playwright/test';
```

Comment:
same here. do tests not pass without this?

Reply:
For all three of these, `check-types` is failing. I can skip the type check in the tests folder but the problem would still exist for the rest of the code